File: cpp/1134.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int alcohol = 0, gas = 0, diesel = 0, n, i;
    cin >> n;

    while (n != 4) {
        switch (n) {
            case 1:
                alcohol++;
                break;
            
            case 2:
                gas++;
                break;

            case 3:
                diesel++;
                break;
        }

        cin >> n;
    }

    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", alcohol);
    printf("Gasolina: %d\n", gas);
    printf("Diesel: %d\n", diesel);
    return 0;
}

File: py/1134.py
fuel = []
while True:
    n = int(input())
    
    if n == 4:
        break
    
    if n >= 1 and n <= 4:
        fuel.append(n)
        
print('MUITO OBRIGADO')
print(f'Alcool: {fuel.count(1)}')
print(f'Gasolina: {fuel.count(2)}')
print(f'Diesel: {fuel.count(3)}')